wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/workflow.html

```diff
@@ -3,23 +3,28 @@
 
 {% comment %}
     This template is used to show Live, Draft, Live and Draft, In Moderation or Live and In Moderation.
-    Sometimes {{ block.super }} will be called two times in the instances where the page is in multiple states eg. Live + Draft
+    Sometimes {{ block.super }} will be called two times in the instances where the object is in multiple states e.g. Live + Draft
 {% endcomment %}
 
 {% block content %}
     <div class="w-space-y-3">
-        {% trans 'Page status: ' as screen_reader_title_prefix %}
-        {% with workflow_state=page.current_workflow_state draft_revision=page.get_latest_revision live_revision=page.live_revision %}
+        {% if page %}
+            {% trans 'Page status: ' as screen_reader_title_prefix %}
+        {% else %}
+            {% trans 'Object status: ' as screen_reader_title_prefix %}
```

Comment:
I wonder if we should drop "Object" from these sentences? I don't think we use this term anywhere else in the UI do we?

Reply:
I agree, I think it's unusual to mention "object" in the UI. I'll change this, thanks!